src/attributes/queries.ts

```diff
@@ -14,9 +15,24 @@ import { AttributeList, AttributeListVariables } from "./types/AttributeList";
 
 const attributeDetails = gql`
   ${attributeDetailsFragment}
-  query AttributeDetails($id: ID!) {
+  ${attributeValueListFragment}
+  query AttributeDetails(
+    $id: ID!
+    $firstValues: Int
+    $afterValues: String
+    $lastValues: Int
+    $beforeValues: String
+  ) {
     attribute(id: $id) {
       ...AttributeDetailsFragment
+      values(
+        first: $firstValues
+        after: $afterValues
+        last: $lastValues
+        before: $beforeValues
+      ) {
+        ...AttributeValueListFragment
+      }
```

Comment:
I'm wondering if extracting attribute value list to separate query would be overengineering things. The idea to fetch only value list while paginating and not details, because it's kinda pointless to update it every time somebody clicks 'prev' or 'next. But it's just an idea.